doc/proposal/in-progress/07-signing.md

```diff
@@ -0,0 +1,230 @@
+
+Example (pretty printed for readability): *TODO*: update
+```
+[
+  {
+    "component": [
+      {
+        "componentReferences": [
+          [
+            {
+              "componentName": "compRefNameComponentName"
+            },
+            {
+              "digest": [
+                {
+                  "hashAlgorithm": "sha256"
+                },
+                {
+                  "normalisationAlgorithm": "jsonNormalisation/V1"
+                },
+                {
+                  "value": "00000000000000"
+                }
+              ]
+            },
+            {
+              "extraIdentity": [
+                {
+                  "refKey": "refName"
+                }
+              ]
+            },
+            {
+              "name": "compRefName"
+            },
+            {
+              "version": "v0.0.2compRef"
+            }
+          ]
+        ]
+      },
+      {
+        "name": "CD-Name"
+      },
+      {
+        "resources": [
+          [
+            {
+              "digest": [
+                {
+                  "hashAlgorithm": "sha256"
+                },
+                {
+                  "normalisationAlgorithm": "manifestDigest/V1"
+                },
+                {
+                  "value": "00000000000000"
+                }
+              ]
+            },
+            {
+              "extraIdentity": [
+                {
+                  "key": "value"
+                }
+              ]
+            },
+            {
+              "name": "Resource1"
+            },
+            {
+              "version": "v0.0.3resource"
+            }
+          ]
+        ]
+      },
+      {
+        "version": "v0.0.1"
+      }
+    ]
+  },
+  {
+    "meta": [
+      {
+        "schemaVersion": "v2"
+      }
+    ]
+  }
+]
+
+```
+
+### Digester
+
+As described, resources have a digest field to store the content hash. Different resource types will use a different normalisationAlgorithm: 
+
+ - `ociArtifactDigest/v1`: uses the hash of the manifest of an oci artifact
+ - `genericBlobDigest/v1`: uses the hash of the blob
+
+## Sign Algorithm
+
+Signing a component-descriptor requires a hash of the normalised component-descriptor, that can be signed.
+The digest fields *MUST* be calculated during normalisation and already existing digest fields *CAN NOT* be trusted. Resources have to be accessed and digested. ComponentReferences has to be followed recursively, calculating the digest for the referenced componentDescriptor. If digests fields for resources or component-references exist, they have to be compared against the calculated digest and rejected if different.
+
+### RSA
+
+After the digest for the normalised component-descriptor is calculated, it can be signed using RSASSA-PKCS1-V1_5 as signature.algorithm. The corresponding signature is stored hex encoded in signature.value with a mediaType of application/vnd.ocm.signature.rsa.
+
+### Signing Server
```

Comment:
I would not add this chapter to the spec. There will be several ways to sign and the component cli is just one tool, making signing possible in a particular environment.

Reply:
agree, chapter is removed.